packages/react/src/context/slash-id-context.tsx

```diff
@@ -243,7 +310,17 @@ export const SlashIDProvider = ({
       return;
     }
 
-    storageRef.current?.removeItem(STORAGE_TOKEN_KEY);
+    // search storage for user tokens
+    Object.entries(storageRef.current!).forEach(([key, maybeToken]) => {
+      // check if the storage entry is a user token
+      if (key.startsWith(LEGACY_STORAGE_TOKEN_KEY)) {
```

Comment:
Wouldn't this remove the new tokens too?

Reply:
yes, that's the point here - remove all the tokens we have in storage for log out